### ISSUE TITLE
Add first-class analytics

### ISSUE THREAD

**Issue by ryangjchandler** [laravel-filament/blog-starter-kit#6]
We could provide some very simple analytics with the starter-kit too. It would really be a `view_count` column on the post.

What do you think @danharrin?

**Comment by danharrin** [comment]
Yeah, good idea! And make the column sortable, track view count on the dashboard etc.
